lab_1_classify_by_unigrams/main.py

```diff
@@ -167,6 +228,7 @@ def detect_language_advanced(
 
     In case of corrupt input arguments, None is returned
     """
+    return None
```

Comment:
revert, if you are not going to write code here

Reply:
if i left functions like they were, code style check was arguing about unused arguments

Reply:
done and errors didn't appear this time